normandy/control/static/control/js/components/action_forms/HeartbeatForm.jsx

```diff
@@ -0,0 +1,105 @@
+import React from 'react'
+import { reduxForm } from 'redux-form'
+import { _ } from 'underscore'
+import { formatLabel } from '../../utils/formHelpers.js'
+
+
+class SurveyForm extends React.Component {
+  render() {
+    const { selectedSurvey, fields, showDefaults } = this.props;
+    const surveyObject = selectedSurvey || fields.defaults;
+    let headerText = 'Default Survey Values';
+    let containerClass = 'fluid-8';
+    if (selectedSurvey) {
+      headerText = selectedSurvey.title.initialValue || 'New survey';
+      containerClass += ' active';
```

Comment:
We have [classnames](https://www.npmjs.com/package/classnames), which makes conditional classes a little more readable. Maybe it'd help here?